src/config.ts

```diff
@@ -65,72 +67,81 @@ const esSearch = new ElasticSearchService(
     undefined,
     { enableMultiTenancy },
 );
+
 const s3DataService = new S3DataService(dynamoDbDataService, fhirVersion, { enableMultiTenancy });
 
 const OAuthUrl =
     process.env.OAUTH2_DOMAIN_ENDPOINT === '[object Object]' || process.env.OAUTH2_DOMAIN_ENDPOINT === undefined
         ? 'https://OAUTH2.com'
         : process.env.OAUTH2_DOMAIN_ENDPOINT;
 
-export const getFhirConfig = async (): Promise<FhirConfig> => ({
-    configVersion: 1.0,
-    productInfo: {
-        orgName: 'Organization Name',
-    },
-    auth: {
-        authorization: authService,
-        // Used in Capability Statement Generation only
-        strategy: {
-            service: 'OAuth',
-            oauthPolicy: {
-                authorizationEndpoint: `${OAuthUrl}/authorize`,
-                tokenEndpoint: `${OAuthUrl}/token`,
+export const getFhirConfig = async (): Promise<FhirConfig> => {
+    if (process.env.ENABLE_SUBSCRIPTIONS) {
```

Comment:
This won't work as expected. env vars are always strings, and the string `'false'` is truthy

Reply:
Ah! Good catch! Will update.